elements/pfe-navigation/src/pfe-navigation--lightdom.scss

```
@@ -8,6 +8,10 @@
 
 // IE11 styles
 @include browser-query(ie11) {
+    pfe-navigation {
+        font-size: pfe-fetch(FontSize);
```

```suggestion
        font-size: 16px;
```

or to be triple sure it works, i put them closer to the selectors:
```
    pfe-icon + [slot="trigger"] > a {
        @include browser-query(ie11) {
            font-size: 16px;
        }
```

```
pfe-navigation-main [slot="trigger"] > a {
    @include browser-query(ie11) {
        font-size: 16px;
    }
}
```